### PR TITLE
Optimize xargo build times

### PR DESCRIPTION
**Pull request by assafmo**

### CODE REVIEW

File: cosmwasm/packages/wasmi-runtime/Makefile [hunk 1]
```diff
@@ -118,11 +119,11 @@ $(Enclave_EDL_Products): Enclave.edl
 	sgx_edger8r --trusted $^ --search-path $(SGX_SDK)/include --search-path $(CUSTOM_EDL_PATH) --trusted-dir ./
 
 check:
-	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS="-Z force-unstable-if-unmarked" xargo check --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx
+	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS=$(Rust_Flags) xargo check --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx
 
 clippy:
-	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS="-Z force-unstable-if-unmarked -D warnings" xargo clippy --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx
+	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS=$(Rust_Flags) xargo clippy --features "SGX_MODE_$(SGX_MODE)" --target x86_64-unknown-linux-sgx -- -D warnings
```
Comment (Cashmaney) [inline cosmwasm/packages/wasmi-runtime/Makefile:125]: Did you check this works? I tried it and it didn't actually break on warnings

File: Makefile [hunk 2]
```diff
@@ -130,28 +130,28 @@ deb: build-linux
...
+	cp ./packaging_ubuntu/control /tmp/SecretNetwork/deb/DEBIAN/control
+	printf "Version: " >> /tmp/SecretNetwork/deb/DEBIAN/control
+	git describe --tags | tr -d v >> /tmp/SecretNetwork/deb/DEBIAN/control
+	echo "" >> /tmp/SecretNetwork/deb/DEBIAN/control
+	cp ./packaging_ubuntu/postinst /tmp/SecretNetwork/deb/DEBIAN/postinst
+	chmod 755 /tmp/SecretNetwork/deb/DEBIAN/postinst
+	cp ./packaging_ubuntu/postrm /tmp/SecretNetwork/deb/DEBIAN/postrm
+	chmod 755 /tmp/SecretNetwork/deb/DEBIAN/postrm
+	dpkg-deb --build /tmp/SecretNetwork/deb/ .
+	-rm -rf /tmp/SecretNetwork
```
Comment (Cashmaney) [inline Makefile:154]: Wat? This is already renamed in develop.

File: cosmwasm/packages/wasmi-runtime/Makefile (hunk 1 above)
Comment (assafmo) [reply cosmwasm/packages/wasmi-runtime/Makefile:125]: Weird, we tested it and it worked cc @reuvenpo

Comment (Cashmaney) [reply cosmwasm/packages/wasmi-runtime/Makefile:125]: Yeah just checked and it seems to work. Idk, maybe I did something wrong last time I tried

**Pull request merged**